oci/spec_opts.go

```diff
@@ -896,7 +896,9 @@ func WithAppendAdditionalGroups(groups ...string) SpecOpts {
 			}
 			ugroups, err := user.ParseGroupFile(gpath)
 			if err != nil {
-				return err
+				if !os.IsNotExist(err) {
+					return err
+				}
```

Comment:
Perhaps make it a real one-line change;

```go
if err != nil && !os.IsNotExist(err) {
```

(could also use `errors.Is(err, fs.ErrNotExist)` / `errors.Is(err, os.ErrNotExist)`)

Reply:
In other parts of the code `os.IsNotExist(err)` is used, I can switch though if you really want

Reply:
I'm fine with either `os.IsNotExist(err) ` or `errors.Is()`; ISTR `os.IsNotExist` also had some code to handle some corner-cases for cgroups (which isn't handled by `errors.Is`), so changing it in other places would depend on what it's used for. 

Was mostly looking at the "make it a one-line", as it slightly reduces cognitive overload (in already complex code). Not an absolute dealbreaker, but as we're changing it 🤷‍♂️